decidim-elections/app/views/decidim/elections/elections/_election_aside.html.erb

```diff
@@ -0,0 +1,7 @@
+<section class="layout-aside__section">
+  <div class="election__aside-vote layout-aside__ctas-buttons" data-sticky-buttons>
```

Comment:
After you merge with develop, you can use the work done in #15064

```suggestion
  <div class="election__aside-vote layout-aside__ctas-buttons" data-controller="sticky-buttons">
```

Reply:
ok!